From d17eec7fa93431a9aabac7296afe330370e82e40 Mon Sep 17 00:00:00 2001
From: Vadym Kochan <vadym.kochan@plvision.eu>
Date: Mon, 24 Feb 2020 00:07:55 +0200
Subject: [PATCH] package/libyang: enable LYD_PRIV option

This option is required by frr package, so enable it by default.

Signed-off-by: Vadym Kochan <vadym.kochan@plvision.eu>
---
 package/libyang/libyang.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/package/libyang/libyang.mk b/package/libyang/libyang.mk
index cfec08c..9bddcaf 100644
--- a/package/libyang/libyang.mk
+++ b/package/libyang/libyang.mk
@@ -15,7 +15,8 @@ HOST_LIBYANG_DEPENDENCIES = host-pcre
 
 LIBYANG_CONF_OPTS = \
 	-DENABLE_VALGRIND_TESTS=OFF \
-	-DGEN_PYTHON_BINDINGS=OFF
+	-DGEN_PYTHON_BINDINGS=OFF \
+	-DENABLE_LYD_PRIV=ON
 
 HOST_LIBYANG_CONF_OPTS = \
 	-DENABLE_VALGRIND_TESTS=OFF \
-- 
2.17.1

